### PR TITLE
Ability to set map coordinate precision

### PR DESCRIPTION
**Pull request by sinkda**
As suggested by user Shelfen on Discord.  Ability to choose the decimal precision for the coordinates of player and cursor on the map (not minimap due to space limitations)

### CODE REVIEW

File: Questie.lua [hunk 1]
```diff
@@ -623,10 +638,10 @@ local options = {
 					name = "Reset Questie",
 				},
 				Spacer_B = _QuestieOptions:Spacer(12),
-				github_text = {
+				reset_text = {
```
Comment (wallrik) [inline Questie.lua]: Is this change intended for this pull request?

Comment (sinkda) [reply Questie.lua]: Yes this change is intended as it's a copy/paste error from the original commit.  The table key for "GitHub_text" is already used below so it was ts that texnot showing this text.  by changing it to "reset_text" it let show and not be overwritten.
